godot-kotlin/godot-library/src/nativeGen/kotlin/godot/Texture.kt

```diff
@@ -1,5 +0,0 @@
-package godot
-
-//TODO: this needs to be properly implemented. It's just here for the implementation of the PropertyTypeHintAnnotations
-open class Texture : Resource() {
```

Comment:
Because this was deleted, the sample cannot build atm. Either we restore this file or remove the corresponding testing lines in the test class inside the samples